src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -547,6 +547,12 @@ impl ToQemuParams for Machine {
         if let Some(mem_backend) = &self.memory_backend {
             params.push(format!("memory-backend={}", mem_backend));
         }
+
+        #[cfg(target_arch = "aarch64")]
+        params.push(format!("usb=off"));
+        #[cfg(target_arch = "aarch64")]
+        params.push(format!("gic-version=3"));
```

Comment:
This should be gic-version=host so that we always use what is available on the host.